frontend/src/components/forms/CreateLog.tsx

```diff
@@ -147,8 +208,27 @@ const CreateLog = () => {
     setNotesError(inputValue === "");
   };
 
-  const [isCreateOpen, setCreateOpen] = React.useState(false);
+  // fetch resident + employee data for log creation
+  const getLogEntryOptions = async () => {
+    const residentsData = await ResidentAPIClient.getResidents(true, 1, 1)
+
+    if (residentsData && residentsData.residents.length !== 0) {
+      // TODO: Remove the type assertions here
```

Comment:
Could you clarify this?

Reply:
We're leveraging non-null type assertions on those properties (below the comment IIRC) that we should probably refactor / remove at some point ; I included them for now to a) make my life easier with some of the merge conflicts and b) workaround to speed this up ; I've already made an issue to track this here - https://github.com/uwblueprint/supportive-housing/issues/148